spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/resource/about/RuntimeEnvironment.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.rest.resource.about;
+
+/**
+ * Provides information about the runtime environment.
+ *
+ * @author Gunnar Hillert
+ */
+public class RuntimeEnvironment {
+
+	/**
+	 * Default constructor for serialization frameworks.
+	 */
+	public RuntimeEnvironment() {
+	}
+
+	/**
+	 * The {@link RuntimeEnvironmentDetails} of the deployer.
+	 */
+	private RuntimeEnvironmentDetails deployer;
```

Comment:
Should read `appDeployer` rather than just `deployer`, to be consistent with `taskLauncher`